cfy_manager/components/postgresql_server/config/patroni.conf

```diff
@@ -0,0 +1,65 @@
+scope: postgres
+namespace: /db/
+name: pg{{ manager.private_ip.replace('.', '_') }}
```

Comment:
consider using manager.hostname instead of manager.private_ip

Reply:
All the certs are done by IP, and it won't cope with the IP changing so I think it probably makes more sense to use the IP?